drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -29,7 +28,10 @@ static void set_spare_desc_q_map(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 {
 	unsigned short spare_desc_indx = 0;
 
-	__ASSERT(fmac_dev_ctx->fpriv->num_tx_tokens_per_ac != 0, "num_tx_tokens_per_ac is zero");
+	wifi_nrf_osal_assert(fmac_dev_ctx->fpriv->opriv,
```

Comment:
Commit log:

```
drivers: wifi:  Add OS agnostic assertion support

This abstracts assertions to be used in OSAL code rather than
directly invoking OS specific assertions.
```